### DISCUSSION TITLE
Zooming shortcuts

### DISCUSSION THREAD

**Discussion by exanrauzer** [Zettlr/Zettlr#4448] [Q&A]
Striking the iron while it's hot, I wanted to ask about something that's been bothering me since ever:

Why is it that to zoom out, one only needs to use Ctrl and the minus key, but to zoom in, Shift is required? In all the applications I'm familiar with, Ctrl "-", and Ctrl "=" are the standard shortcuts for zooming.

Continuing with this topic, I would suggest the following mechanism instead: Ctrl "-" and Ctrl "=" would zoom the text, while Ctrl + Shift "-" and Ctrl + Shift "=" would zoom the interface.

Personally, I frequently use zoom functionality. I enjoy adjusting the font size according to my current preference. I also often minimize the font size significantly when people are hanging around behind my back;)

_Not to mention ctrl + (shift +) scroll feature!_

**Comment by exanrauzer** [comment]
@nathanlesage I can't believe you're turning a blind eye to this aspect. It's as obvious as a penguin in a desert;)

**Reply by nathanlesage** [reply]
These are minuscule issues and furthermore not even really fixable since it interferes with the keyboard layout. If it is important to you, please feel free to tinker with it yourself! I cannot do anything about this before 3.0. I am extremely stretched thin and fully loaded with all the other issues you've opened.

**Reply by exanrauzer** [reply]
Oh, I'm just curious! I'd like to know the origin. After all, it's a discussion section, right?

Please, do not consider my concerns as important. Since it's your software, you know best what the priorities are. Issues are meant for helping you ensure everything is covered and running smoothly. Don't shoot the messenger!

**Reply by nathanlesage** [reply]
It's a discussion threat but it's an issue, and every notification is just more work for me 😪
